### ISSUE TITLE
Feature Parity for the DASH-HLS Interop Spec

### ISSUE THREAD

**Issue by johnsim** [cta-wave/content-specification-task-force#43]
Media Segment portability between DASH and HLS is a must, but do we also require feature parity? And if so, assuming this is not complete feature parity, what are the features which must have equivalent mappings between DASH and HLS?

**Comment by Yasser-Comcast** [comment]
So while we may not convert between different types of manifests, there are use cases that may demand a knowledge of level of parity between these types of manifests. Use cases should consider ad insertion cases, low latency cases, live cases. I believe ad insertion is probably the trickiest in both the linear and VOD cases since in the dash sense the makeup of different period can change. In terms of feature parity and the constraints that go along with it could be categorized by duration ( time equivalency), representation ( quality), adaptation ( media component), and Period including gap signaling. IS it enough to playout the same content? Or do I always have to have 4K with eng/Spanish/Descriptive video in either type of manifests, can I use 3rd party ads or do I need to format those ads along with the main content.

**Comment by haudiobe** [comment]
I believe we should express the whole problem as use cases and based on the use cases, derive the requirements.

**Comment by technogeek00** [comment]
Another second for defining the problems here as use cases, it's worked well in the path to drive requirements and the resulting descriptions will likely fulfill both manifest generation and rewrite cases.

One recommendation I would make is that we clearly separate use cases that are attempting to represent one presentation from multiple presentations. The basics of representing a singular presentation and its various features in DASH and HLS will bring in the majority of the constructs and will be the most straight-forward. Bringing in multiple presentations, potentially interleaved, is where the most significant divergence between these two formats occur and will take the most study, feedback, and possibly constraints.

**Comment by johnsim** [comment]
I agree that use cases should be use cases, deriving requirements. I also agree we should focus on single period models first. It was suggested in that we derive domains of feature parity based on use cases.
